config/Config-kernel.in

```diff
@@ -629,6 +629,9 @@ config KERNEL_DEBUG_GPIO
 	bool "Compile the kernel with gpio debugging"
 	select KERNEL_DEBUG_KERNEL
 
+config KERNEL_CHECKPOINT_RESTORE
+  bool
+
```

Comment:
Does lxc need this option? Is it needed for lxc at all or only for the lxc-checkpoint feature?
If only a package from the package feed selects it, it will not be build in the default images by buildbot, because it only builds the image default packages.

If we really need it you cold activate it for not small:
```suggestion
config KERNEL_CHECKPOINT_RESTORE
  bool "Compile the kernel with checkpoint/restore support"
  default y if !SMALL_FLASH

```

Reply:
AFAIK it is only needed for `lxc-checkpointrestore`. I dropped it from this PR and will modify https://github.com/openwrt/packages/pull/25719

Good feedback as always @hauke